src/fdk_rdf_parser/reference_data/extend_cpsvno_service.py

```diff
@@ -175,8 +174,10 @@ def extend_participations(
         if participation.role:
             for role in participation.role:
                 extended_role = extend_reference_data_code(role, ref_data.role_types)
-                extended_role = cast(ReferenceDataCode, extended_role)
                 extended_roles.append(extended_role)
```

Comment:
bittelitt overkill med den variabelen nå
```suggestion
                extended_roles.append(extend_reference_data_code(role, ref_data.role_types))
```